.github/workflows/publish-latest.yml

```diff
@@ -22,11 +22,46 @@ jobs:
         id: has-changesets
         uses: ./.github/actions/has-changesets
 
-  publish:
+  build-test:
     runs-on: ubuntu-latest
-    environment: ci
     needs: setup
-    if: ${{ needs.setup.outputs.has-changesets != 'true' }}
+    # if: ${{ needs.setup.outputs.has-changesets != 'true' }}
```

Comment:
Do we need this commented out code?

Reply:
Ooh, I was just POCing this by doing a test run it on this PR. I commented it out so I don't accidentally publish anything right now ;)

Reply:
Will revert before I mark this PR ready for review